backend/app/controllers/api/v1/environment_controller.rb

```diff
@@ -14,7 +15,8 @@ def index
       homepage_url: Rails.application.secrets.homepage_url,
       banners_schedule: Banner.active.to_a || [],
       rewards_schedule: Reward.all.to_a,
-      is_impersonating: session[:impersonating].present?
+      is_impersonating: session[:impersonating].present?,
+      is_country_eligible: country.blank? || Kinetic::ELIGIBLE_COUNTRY_CODES.include?(country)
     )
```

Comment:
we should extract the eligibility check in the eligibility_controller to a central location so that it's identical.   maybe an services/eligibility function that you pass the request headers to?